ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/TableRowsEntityFactory.java

```diff
@@ -88,7 +89,17 @@ private static LogicalSchema addWindowFieldsIntoSchema(
       window.end().map(Instant::toEpochMilli).ifPresent(builder::add);
     });
 
-    builder.addAll(row.value().getColumns());
+    for (Object col: row.value().getColumns()) {
```

Comment:
talked offline - I think a "better" fix is not to coerce to string but rather not use `ImmutableList` here, which doesn't allow nulls.